apps/ejabberd/src/ejabberd_sm.erl

```diff
@@ -744,7 +746,7 @@ route_message(From, To, Packet) ->
               %% Route messages to all priority that equals the max, if
               %% positive
               fun({Prio, Pid}) when Prio == Priority ->
-                      % we will lose message if PID is not alive
+                                                % we will lose message if PID is not alive
```

Comment:
Yeah... My formatter (emacs) assumes that comments with single '%' have fixed indentation because of reasons. I'm gonna revert all those changes.